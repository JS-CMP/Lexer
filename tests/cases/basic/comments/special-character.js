// Comment with special characters #!@*$
var e = "test" /* end */