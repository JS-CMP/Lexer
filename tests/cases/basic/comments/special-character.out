#include "types/JsAny.hpp"
#include "global/global.hpp"
#include "types/objects/Types.hpp"

int main() {JS::Any e=JS::Any("test");}
